.github/workflows/svc-bip-api-integration-test.yml

```diff
@@ -44,14 +44,13 @@ jobs:
           export -p | sed 's/declare -x //'
 
           ./gradlew :dockerComposeUp
+          ./gradlew -p mocks :build
```

Comment:
Looks like this guy is still hanging around

Reply:
addressed